nim/scorper/server.nim

```diff
@@ -0,0 +1,16 @@
+import strutils
+import scorper
+
+proc cb(req: Request) {.async.} =
+  if req.meth == HttpGet:
+    if req.path == "/":
+      await req.resp("")
+    elif req.path.startsWith("/user/"):
+      let id = req.path[6 .. ^1]
+      await req.resp(id)
+  elif req.meth == HttpPost:
+    if req.path == "/user":
+      await req.resp("")
+
+const address = "127.0.0.1:3000"
```

Comment:
please use broadcast address (otherwise we could not connect to the app since inside a container)
```suggestion
const address = "0.0.0.0:3000"
```